10-hypothesis-testing.Rmd

```diff
@@ -253,7 +253,12 @@ knitr::include_graphics("images/sampling/promotions/shared_spreadsheet.png")
 
 ```{r, eval=TRUE, echo=FALSE, message=FALSE, warning=FALSE}
 # https://docs.google.com/spreadsheets/d/1Q-ENy3o5IrpJshJ7gn3hJ5A0TOWV2AZrKNHMsshQtiE/edit#gid=0
-shuffled_data <- read_csv("https://docs.google.com/spreadsheets/d/e/2PACX-1vQXLJxwSp1ALEJ1JRNn3o8K3jVdqRG_5yxpoOhIFYflbFIkb2ttH73w8mljptn12CsDyIvjr5p0IGUe/pub?gid=0&single=true&output=csv")
+if(!file.exists("rds/shuffled_data.rds")){
```

Comment:
yeah, this is fine. but since the update will no longer be dynamic, I've added the following release step: Delete contents of `rds/` and rebuild all `.rds` files in case any got stale.